### PR TITLE
chore: update to aspect_bazel_lib 1.2.0

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

**Comment by alexeagle** [comment]
we have to push new bazel-lib to our own BCR fork to make this green. @kormide is working on the automation, maybe we can get that across a finish line

**Comment by kormide** [comment]
> we have to push new bazel-lib to our own BCR fork to make this green. @kormide is working on the automation, maybe we can get that across a finish line

The automation exists to PR against the real BCR. There's just an issue with my PAT being associated to my personal email and failing their CLA.

I suppose it makes sense to also automate pushing to our fork.

**Comment by gregmagolan** [comment]
> Here's the PR to our bcr: [aspect-build/bazel-central-registry#7](https://github.com/aspect-build/bazel-central-registry/pull/7)

🙏

File: js/repositories.bzl [hunk 1]
```diff
@@ -8,7 +8,6 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
 versions = struct(
-    aspect_bazel_lib = "1.1.0",
```
Comment (kormide) [inline js/repositories.bzl:11]: Not using this anymore?

Comment (gregmagolan) [reply js/repositories.bzl:11]: there is a mix of both styles across our repos but this one is the outlier so just going with the standard style now

**Pull request merged**